.github/workflows/test.yml

```
@@ -101,6 +101,50 @@ jobs:
           ./kronos examples/functions.kr
           echo "✓ Example files run successfully"
 
+  lsp-tests:
+    name: LSP Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)